includes/helpers/class-http.php

```
@@ -19,18 +19,18 @@ public function get( $url, $args = array() ) {
             return $this->handle_wp_error( $response, $url, $args );
         }
 
-        $response_code = wp_remote_retrieve_response_code( $response );
+        $response_code    = wp_remote_retrieve_response_code( $response );
         $response_message = wp_remote_retrieve_response_message( $response );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$response_message'. (undefined)

(UnusedLocalVariable)